pyvrp/cpp/crossover/crossover.cpp

```diff
@@ -117,13 +115,13 @@ void pyvrp::crossover::greedyRepair(Routes &routes,
 
             if (idx == 0)  // try after depot
             {
-                prev = 0;
+                prev = 0;  // TODO hardcoded depot
                 next = route[0];
             }
             else if (idx == route.size())  // try before depot
             {
                 prev = route.back();
-                next = 0;
+                next = 0;  // TODO hardcoded depot
```

Comment:
I don't have a good solution to this yet so I suggest we table this until the PR that actually addresses multiple depots.